packages/web_drivers/README.md

```diff
@@ -1,23 +1,19 @@
-# To install and start Chrome Driver.
+**To install and start Chrome Driver.**
 
-dart lib/web_driver_installer.dart chromedriver &
+`dart lib/web_driver_installer.dart chromedriver &`
```

Comment:
Everywhere else we use triple backticks for non-inline code snippets ([example](https://github.com/flutter/flutter/blob/master/dev/devicelab/README.md))

Reply:
BTW, do we need the `&` at the end? It prevents the ability to use Ctrl + C to shutdown the driver. I'm sure developers can consult with StackOverflow or what not to learn how to run a process in the background.

Reply:
> Everywhere else we use triple backticks for non-inline code snippets ([example](https://github.com/flutter/flutter/blob/master/dev/devicelab/README.md))

good idea.